odc-index/requirements.txt

```diff
@@ -1,6 +1,7 @@
 # s3/boto, let aiobotocore decide botocore/boto3/awscli versions
 aiobotocore[boto3,awscli]==0.11.0
 python-dateutil==2.8.0
+thredds_crawler
```

Comment:
Are you using it directly or via `s3-find`? Because line below should install `thredds_crawler` dependency already

`odc-apps-cloud[GCP,THREDDS]`

Reply:
meant to say `thredds-to-tar` instead of `s3-find`